app/components/ui/Navigation/Navigation.tsx

```diff
@@ -25,6 +27,16 @@ export const Navigation = () => {
     extractNavigationMenusFromNavigationResponse(navigationResponse);
   const mobileSubMenuIsActive = !!activeMobileSubMenu;
 
+  const location = useLocation();
+  const hiddenInputRef = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    if (hiddenInputRef.current) {
+      hiddenInputRef.current.focus();
+      hiddenInputRef.current.blur();
+    }
+  }, [location]);
```

Comment:
This code seems redundant, is this supposed to be here?

Reply:
removed!